docs/reference/mapping/fields/routing-field.asciidoc

```diff
@@ -5,15 +5,6 @@ The routing field allows to control the `_routing` aspect when indexing
 data and explicit routing control is required.
 
 [float]
-==== store / index
-
-The first thing the `_routing` mapping does is to store the routing
-value provided (`store` set to `true`) and index it (`index` set to
-`not_analyzed`). The reason why the routing is stored by default is so
-reindexing data will be possible if the routing value is completely
-external and not part of the docs.
-
-[float]
```

Comment:
In the next para, delete "or derived from the doc".  

Also, need to check with @martijnvg, but I think you can delete "A delete operation will be broadcasted to all shards if no routing value is provided and `_routing` is required."